src/main/java/net/fabricmc/loader/impl/FabricLoaderImpl.java

```diff
@@ -282,6 +283,17 @@ private void setup() throws ModResolutionException {
 		modCandidates = null;
 	}
 
+	private void dumpNonFabricMods(List<Path> nonFabricMods) {
+		StringBuilder outputText = new StringBuilder();
+
+		for (Path nonFabricMod : nonFabricMods) {
+			outputText.append("\n\t- ").append(nonFabricMod.getFileName());
```

Comment:
Question: Whats the behaviour when a mod has a none fabric mod contained within?

E.g you have `mods/mymod.jar/notamod.jar` where mymod.jar is a valid fabric mod, and within its fabric.mod.json file it points to `nodamod.jar`

I think it might be confusing for it to print out that it found a none mod jar `notamod.jar`

Reply:
It only adds to the non-fabric-mod list from ModScanTask::computeJarFile, so it will ignore any embedded mods as well as directory mods which do not seem to load anyways.

Reply:
Great thanks.